### ISSUE TITLE
Continuous zoom is broken with a pointing device (mouse, trackpad etc.)

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#4314]
## Package
lib-classifier
## Describe the bug
Zooming in and out only changes the zoom level by one step when you click the zoom in/out buttons. Continuous zoom from the keyboard with +/- isn't affected.

## To Reproduce
1. Go to https://frontend.preview.zooniverse.org/projects/pmlogan/poets-and-lovers/classify/workflow/21362/subject-set/104809
2. Click on the Zoom In button

## Expected behavior
Zooming in and out should be continuous while the pointer is down.

As a workaround, you can zoom with the +/- keys then pan the subject with the cursor keys.
